woocommerce/wc-admin
# Removing grow section from local free extensions in OBW

Fixes #7312

Removing the grow section, and all plugins, from the local free extensions on the OBW as they're being moved to the new marketing task.

Note that this only impacts local extensions, whereas by default most will be using remote free extensions that are hosted on WCCOM. That will require another PR in that repo.

Note that I deleted entirely the `client/profile-wizard/steps/business-details/data/extension-benefits.js` file, which just seemed to be orphaned. Let me know if this is actually utilized in some way and I can restore it.

## Tasks
- [x] Remove the "Grow" extensions from the Business Details step of the OBW
- [x] Remove the "Get the basics" header

### Screenshots

![image](https://user-images.githubusercontent.com/444632/127049700-303e8f6d-b4cd-4748-bd09-0f019f9d3229.png)


### Detailed test instructions:

1. Ensure that the `remote-extensions-list` feature is toggled **off**
2. Navigate through the OBW until you reach the recommended business features list, and confirm that the "Grow your store" section along with related plugins are no longer displayed, as pictured.
3. Also ensure that "Get the basics" header is no longer displayed
4. Hit continue, and then confirm that only the listed plugins were installed.
5. Repeat steps 2-4 after reenabling `remote-extensions-list` feature and refreshing.
